src/estimation/estimation_main.py

```diff
@@ -35,6 +36,9 @@ def run_estimation(
 
     NETWORK_OR_HDFS = config["global"]["network_or_hdfs"]
     est_path = config[f"{NETWORK_OR_HDFS}_paths"]["estimation_path"]
+    col = config["global"]["duplicate_check_col"]
+
```

Comment:
We don't need this in the config, just call the function like this: `duplicates_check(df,  "reference")`

Reply:
removed and added in 'reference'